### ISSUE TITLE
BackPack Deleting

### ISSUE THREAD

**Issue by Reninsane** [FTBTeam/FTB-Modpack-Issues#5599]
### Modpack

FTB NeoTech

### Modpack version

1.6.0

### Has the pack been modified

No

### Describe the bug

When logging out and back in with a backpack on or in inv, Backpack and all contents deletes itself, Tested this over previous versions and this is the 4th backpack iv lost

### Steps to reproduce

1. Wear Backpack
2. Log out
3. Log in
4. Backpacks gone

### Expected behaviour

Keeping my backpack

**Comment by Gaz492** [comment]
I have not been able to reproduce this in my test world, Is this happening every time you log out and back in?
Does this happen with an empty backpack or when you have items/upgrades in them.

**Comment by Reninsane** [comment]
Iv tried all, With and without upgrades, Only fix iv found for it at the moment, Is placing the backpack down just before i log out. It is happening every single time i log in and out, Tried adding it to bauble slot, Hotbar, or just in inv. Its weird

**Comment by Saereth** [comment]
I tried this on a seperate server instance and fresh sp game as well and definitely cant produce this, so weird

**Comment by ArtpokeBr** [comment]
Yeah, Definetly something weird, but I also couldn't reproduce this, so I'll be closing this issue.
